awsx/schema-types.ts

```diff
@@ -118,6 +119,20 @@ export interface ImageArgs {
     readonly repositoryUrl: pulumi.Input<string>;
     readonly target?: pulumi.Input<string>;
 }
+export abstract class RegistryImage<TData = any> extends (pulumi.ComponentResource)<TData> {
+    public image!: unknown | pulumi.Output<unknown>;
```

Comment:
What is `unknown` here exactly?

Reply:
Uh that's interesting, good catch!
Seems like the script for generating provider internal types has a bug or cannot properly handle references to docker??

The SDK and schema look good, but I'm gonna dig into this

Reply:
Thanks!!

Reply:
Yeah, the type-gen for provider internal types was only configured to handle aws. I've added docker as well now